test/tools/libtesteth/TestSuite.cpp

```diff
@@ -98,6 +98,7 @@ void addClientInfo(json_spirit::mValue& _v, fs::path const& _testSource, h256 co
 
 		clientinfo["filledwith"] = test::prepareVersionString();
 		clientinfo["lllcversion"] = test::prepareLLLCVersionString();
+		clientinfo["binaryenVersion"] = test::prepareBinaryenVersionString();
```

Comment:
please explain more who binaryen works and how it's related to evm.

Reply:
It used for ewasm tests. Binaryen is used in testeth for at least 10 months.

Reply:
could not this information somehow be retrived from aleth version ?

Reply:
It could right now. However I've stated the goal in the description of the PR:
> Will create a PR next which moves binaryen into an external binary (just like lllc).

Basically in the future it shouldn't have a built in binaryen, just rely on an external tool.